src/com/palmergames/bukkit/towny/permissions/PermissionNodes.java

```diff
@@ -274,6 +274,8 @@
 		TOWNY_COMMAND_TOWNYADMIN_NATION_SET("towny.command.townyadmin.nation.set"),
 		TOWNY_COMMAND_TOWNYADMIN_NATION_TOGGLE("towny.command.townyadmin.nation.toggle"),
 		TOWNY_COMMAND_TOWNYADMIN_NATION_BANKHISTORY("towny.command.townyadmin.nation.bankhistory"),
+		TOWNY_COMMAND_TOWNYADMIN_NATION_ENEMY("towny.command.townyadmin.nation.enemy"),
+		TOWNY_COMMAND_TOWNYADMIN_NATION_ALLY("towny.command.townyadmin.nation.ally"),

```

Comment:
I'm sure I'm missing it, but I can't see where these are used?

Reply:
These two get picked up in the catch-all node test: https://github.com/TownyAdvanced/Towny/blob/233358d6653c63b80e2fbd4f3eba2c9ccc28e1ce/src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java#L1250